### PR TITLE
 Add support for Deny Assignments

### PR DESCRIPTION
**Pull request by kirankulkarni747**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by kirankulkarni747** [comment]
Link to REST spec PR - https://github.com/Azure/azure-rest-api-specs/pull/3987

File: src/SDKs/Authorization/Management.Authorization/Microsoft.Azure.Management.Authorization.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<Description>Provides capabilities to query Microsoft Azure Management Authorization.</Description>
 		<AssemblyTitle>Authorization Library for Microsoft Azure Management Authorization</AssemblyTitle>
-		<Version>2.8.0-preview</Version>
+		<Version>2.9.0-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.Authorization</AssemblyName>
 		<PackageId>Microsoft.Azure.Management.Authorization</PackageId>
 		<PackageTags>Microsoft Azure Management;Authorization;</PackageTags>
```
Comment (dsgouda) [inline src/SDKs/Authorization/Management.Authorization/Microsoft.Azure.Management.Authorization.csproj]: Please update the `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Pull request merged**